exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,38 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number <= 0) {
```

Comment:
Since 0 is not negative (or positive) it shouldn't pass as true. So this line should be `if (number < 0)`